net/natmap/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+"$1" 2>&1 >/dev/null | grep "$2"
```

Comment:
`>/dev/null` should be dropped, or stdout will get nothing

Reply:
FIxed.